lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py

```diff
@@ -0,0 +1,566 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sqldatabase
+version_added: "2.5"
+short_description: Manage SQL Database instance.
+description:
+    - Create, update and delete instance of SQL Database.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database to be operated on (updated or created).
+        required: True
+    location:
+        description:
+            - Resource location. If not set, location from the resource group will be used as C(default).
+    collation:
+        description:
+            - The collation of the database. If I(create_mode) is not C(default), this value is ignored.
+    create_mode:
+        description:
+            - Specifies the mode of database creation.
+            - "C(default): regular database creation."
+            - "C(copy): creates a database as a C(copy) of an existing database. I(source_database_id) must be specified as the resource ID of the source dat
```

Comment:
Don't put a newline in the middle of a work, put the newline before the work actually starts. There are a few examples of that in the module.